<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>1000</width><height>640</height></rect>
  </property>
  <property name="windowTitle">
   <string>Aplikasi DB Hukum</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect><x>0</x><y>0</y><width>1000</width><height>22</height></rect>
   </property>
   <widget class="QMenu" name="menuMaster">
    <property name="title"><string>Master</string></property>
    <addaction name="actionFormJenis"/>
    <addaction name="actionFormRoles"/>
    <addaction name="actionFormUsers"/>
   </widget>
   <widget class="QMenu" name="menuTransaksi">
    <property name="title"><string>Transaksi</string></property>
    <addaction name="actionFormInformasi"/>
    <addaction name="actionFormProduk"/>
   </widget>
   <addaction name="menuMaster"/>
   <addaction name="menuTransaksi"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFormJenis">
   <property name="text"><string>Form Jenis</string></property>
  </action>
  <action name="actionFormRoles">
   <property name="text"><string>Form Roles</string></property>
  </action>
  <action name="actionFormUsers">
   <property name="text"><string>Form Users</string></property>
  </action>
  <action name="actionFormInformasi">
   <property name="text"><string>Form Informasi</string></property>
  </action>
  <action name="actionFormProduk">
   <property name="text"><string>Form Produk</string></property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
